game2.html

```diff
@@ -1,24 +1,20 @@
 <!--How many hours have you lived game-->
-
-
-
 <script type="text/javascript">
-	var age, hours, message;
-
-	age = prompt("How old are you?")
-	hours = prompt("If you are " + age + " then how many hours have you lived?")
-	answer = age * 365 * 24
-	
-	if (hours === answer) {
-		message = "That's right!";
-
-	} else {
-		message = "That's incorrect you've been alive for" + " " + answer + " hours";
-
-	}
-
-	
-	alert(message);
+	var age, hours, answer;
+		age = prompt("How old are you?");
+		hours = prompt("If you are " + age + " then how many hours have you lived?");
+		answer = age * 365 * 24;
+
+	while (hours != answer) {//Should keep prompting while there's no match
+    if (hours < answer) { //guess is lower than answer
+      hours = prompt("Your guess is too low, you should live longer. Guess again.");
+    }
+    else if (hours > answer) {// guess is higher than the answer
+      hours = prompt("You haven't live that long yet. Guess again.");
+    }
+  }
```

Comment:
Nice job getting a while loop in. I will say that the code indenting looked a little better before the revisions. Were gonna be sticklers on that in upcoming assignments.